src/libraries/System.Net.Http/tests/UnitTests/Headers/HttpRequestHeadersTest.cs

```diff
@@ -1455,6 +1455,17 @@ public void CacheControl_UseAddMethod_AddedValueCanBeRetrievedUsingProperty()
             Assert.Equal(value, headers.CacheControl);
         }
 
+        [Fact]
+        public void CacheControl_ValidAndInvalidValues_ReturnValidValue()
+        {
+            headers.TryAddWithoutValidation("Cache-Control", "invalid");
```

Comment:
```suggestion
            headers.TryAddWithoutValidation("Cache-Control", "<invalid>");
```

`"invalid"` is a valid value for Cache-Control. I expect this test will fail after changing it to an invalid token like `<invalid>`.